tests/extended/test_trainer_ext.py

```diff
@@ -193,7 +206,7 @@ def test_trainer_log_level_replica(self, experiment_id):
         self.assertEqual(n_matches, data["n_matches"])
 
     @slow
-    def test_run_seq2seq_slow(self):
+    def test_run_seq2seq(self):
```

Comment:
Why rename this one? Doesn't seem linked to the PR.

Reply:
because we don't use `slow` in the test names, that's what the decorator is for, so I was trying to make it consistent.

The reason I changed it here was because Manuel was mimicking this old test name when creating bnb tests and calling them `_slow`.

Again I can change it in another PR if it's simpler.

Please let me know.

Reply:
Thanks for the explanation, that's fine by me!